plugins/occt/CMakeLists.txt

```diff
@@ -23,12 +23,21 @@ endif()
 f3d_plugin_init()
 
 f3d_plugin_declare_reader(
-  NAME OCCT
-  EXTENSIONS stp step igs iges
-  MIMETYPES model/iges application/vnd.step
+  NAME STEP
+  EXTENSIONS stp step stpnc p21 210
```

Comment:
is that the exact same format as step ?

Reply:
According to the wikipedia and iana page, yes